drf_sobitie/conf/settings.py

```diff
@@ -72,7 +72,7 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
-ROOT_URLCONF = "drf_sobitie.urls"
+ROOT_URLCONF = "conf.urls"
```

Comment:
Тут также предлагаю от корня импорты делать:
```suggestion
ROOT_URLCONF = "drf_sobitie.conf.urls"
```